tools/GENERATE-PYTHON.sh

```diff
@@ -0,0 +1,39 @@
+set -e
+
+# clone registry protos
+REGISTRY_PROTOS_PATH='consistency/registry-protos'
```

Comment:
I guess you're putting these under "consistency" because you're only expecting to use them in that project. That's ok for now, we might want to move them later.

Reply:
Yes, we can always move them later, also this is a temporary solution and we might not need to generate the clients manually once wee have our official ones published.